packages/payload/src/fields/baseFields/timezone/defaultTimezones.ts

```diff
@@ -3,58 +3,58 @@ import type { Timezone } from '../../../config/types.js'
-  { label: '(UTC+09:00) Seoul', value: 'Asia/Seoul' },
-  { label: '(UTC+10:00) Brisbane', value: 'Australia/Brisbane' },
-  { label: '(UTC+10:00) Sydney, Melbourne', value: 'Australia/Sydney' },
-  { label: '(UTC+10:00) Guam, Port Moresby', value: 'Pacific/Guam' },
-  { label: '(UTC+11:00) New Caledonia', value: 'Pacific/Noumea' },
-  { label: '(UTC+12:00) Auckland, Wellington', value: 'Pacific/Auckland' },
-  { label: '(UTC+12:00) Fiji', value: 'Pacific/Fiji' },
+  { label: 'Alaska - America (UTC-09:00)', value: 'America/Anchorage' },
+  { label: 'Almaty - Asia (UTC+06:00)', value: 'Asia/Almaty' },
+  { label: 'Arizona (No DST) - America (UTC-07:00)', value: 'America/Phoenix' },
+  { label: 'Athens, Bucharest - Europe (UTC+02:00)', value: 'Europe/Athens' },
+  { label: 'Auckland, Wellington - Pacific (UTC+12:00)', value: 'Pacific/Auckland' },
+  { label: 'Azores - Atlantic (UTC-01:00)', value: 'Atlantic/Azores' },
+  { label: 'Baku - Asia (UTC+04:00)', value: 'Asia/Baku' },
+  { label: 'Bangkok - Asia (UTC+07:00)', value: 'Asia/Bangkok' },
+  { label: 'Beijing, Shanghai - Asia (UTC+08:00)', value: 'Asia/Shanghai' },
+  { label: 'Berlin, Paris - Europe (UTC+01:00)', value: 'Europe/Berlin' },
+  { label: 'Bogota, Lima, Quito - America (UTC-05:00)', value: 'America/Bogota' },
+  { label: 'Brisbane - Australia (UTC+10:00)', value: 'Australia/Brisbane' },
+  { label: 'Buenos Aires - America (UTC-03:00)', value: 'America/Buenos_Aires' },
+  { label: 'Cairo - Africa (UTC+02:00)', value: 'Africa/Cairo' },
+  { label: 'Cape Verde - Atlantic (UTC-01:00)', value: 'Atlantic/Cape_Verde' },
+  { label: 'Caracas - America (UTC-04:00)', value: 'America/Caracas' },
+  { label: 'Central America - America (UTC-06:00)', value: 'America/Guatemala' },
+  { label: 'Central Time (US & Canada) - America (UTC-06:00)', value: 'America/Chicago' },
+  { label: 'Chennai, Kolkata, Mumbai, New Delhi - Asia (UTC+05:30)', value: 'Asia/Calcutta' },
+  { label: 'Cook Islands - Pacific (UTC-10:00)', value: 'Pacific/Rarotonga' },
+  { label: 'Dhaka - Asia (UTC+06:00)', value: 'Asia/Dhaka' },
+  { label: 'Dubai - Asia (UTC+04:00)', value: 'Asia/Dubai' },
+  { label: 'Eastern Time (US & Canada) - America (UTC-05:00)', value: 'America/New_York' },
+  { label: 'Fiji - Pacific (UTC+12:00)', value: 'Pacific/Fiji' },
+  { label: 'Gambier Islands - Pacific (UTC-09:00)', value: 'Pacific/Gambier' },
+  { label: 'Guam, Port Moresby - Pacific (UTC+10:00)', value: 'Pacific/Guam' },
+  { label: 'Hawaii - Pacific (UTC-10:00)', value: 'Pacific/Honolulu' },
+  { label: 'Islamabad, Karachi - Asia (UTC+05:00)', value: 'Asia/Karachi' },
+  { label: 'Jakarta - Asia (UTC+07:00)', value: 'Asia/Jakarta' },
+  { label: 'Lagos - Africa (UTC+01:00)', value: 'Africa/Lagos' },
+  { label: 'London, Lisbon (GMT) - Europe (UTC+00:00)', value: 'Europe/London' },
+  { label: 'Midway Island, Samoa - Pacific (UTC-11:00)', value: 'Pacific/Midway' },
```

Comment:
Midway Island is not in Samoa, though it does use "Samoa Standard Time" which is not actually used in Apia, Samoa, ironically. I hope that this is illustrating just how important it is to just label a datetime-picker with just the IANA labels: `Pacific/Midway` and `Pacific/Apia` respectively.